core/chains/evm/gas/models.go

```diff
@@ -138,6 +147,8 @@ type EvmEstimator interface {
 	//   - be sorted in order from highest price to lowest price
 	//   - all be of transaction type 0x2
 	BumpDynamicFee(ctx context.Context, original DynamicFee, maxGasPriceWei *assets.Wei, attempts []EvmPriorAttempt) (bumped DynamicFee, err error)
+
+	L1Oracle() rollups.L1Oracle
```

Comment:
Think we should return the pointer here. I probably miss this in the write up. Otherwise we could be dereferencing a nil l1Oracle in the estimator

Reply:
Note: We decided against this after further review